src/uucore/src/lib/features/proc_info.rs

```diff
@@ -315,6 +315,7 @@ fn stat_split(stat: &str) -> Vec<String> {
 }
 
 /// Iterating pid in current system
+#[allow(dead_code)]
```

Comment:
please comment why it is dead code :)

Reply:
At first I just did it mindlessly because clippy was complaining. Now that I take a look at it, it seems that the function is only ever used in the test section below. It's used nowhere else.

Actually, the whole file is used absolutely nowhere, so I'm not sure as of what to do with it.
Do we keep it for maybe using it in the future ? Or just get rid of it ?

Reply:
@sylvestre ping

Reply:
Just add a comment explaining this please :)

Reply:
I've actually moved the `#[allow(dead_code)]` to the root of the file, because warning were still emitted on other functions

Reply:
This file was added in #6483. Why do we need both this version and the version in [procps](https://github.com/uutils/procps/blob/main/src/uu/pgrep/src/process.rs)? I can already see bitrot between the two at a glance.

Reply:
I second the question. @sylvestre any clue ?

Reply:
oh, we should remove it from procps

Reply:
@Krysztal112233 would it be possible for you to do it?

Reply:
Sure, but it might take some time.

Reply:
In the meantime, it might be worth just updating the comment to indicate this intention, so this PR can get merged.

Reply:
Good point. Thanks